fixtures/simple-project/user/bryan.json

```diff
@@ -0,0 +1,13 @@
+{
+   "_id":"661d43bc490d045b8ba04d38",
+   "type":"User",
+   "name":"Bryan H.",
```

Comment:
We talk about `displayName` in other places. Is this essentially what that is? Should we be using a `profile.nickname` as a rule?

Reply:
Bryan Haberberger
  1:36 PM
Maybe?  I think I took this from an example of what a User looks like in the `users` table, and I think that is probably meant to be something like the displayName
[1:37](https://slu-research.slack.com/archives/C06NHHFPY3H/p1728499031597649)
handle

Reply:
Let's commit in the fixtures to something TPEN will always use and recommend for a display name within the application.

If it is always in `profile` then the public can see it, if not, we need to explicitly attach it for non-auth'd requests.

Reply:
Upvote your fav:

Reply:
`name`

Reply:
`nickname`

Reply:
`displayName`

Reply:
`profile.` + above option